### PR TITLE
Added the verbose option to switch verbosity on or off

### PR DESCRIPTION
**Pull request by danielpataki**

### CODE REVIEW

**Comment by felixzapata** [comment]
@danielpataki thanks for the pull request. I will take a look as soon as possible.

**Comment by danielpataki** [comment]
Glad if I could help, thanks for the module :)

File: index.js [hunk 1]
```diff
@@ -29,11 +29,18 @@ function fileExists(filePath) {
...
 
   var defaultOptions = {
-    poDel: false
+    poDel: false,
+    verbose: true
   };
 
   var options = customOptions ? R.merge(defaultOptions, customOptions) : defaultOptions;
 
+  function consoleLog(message) {
```
Comment (felixzapata) [inline index.js]: can you replace `consoleLog` with `log`?. I think it will be less confusing

**Comment by danielpataki** [comment]
Sure :) I used that naming just in case you wanted to use `log` for something else down the line.

**Comment by felixzapata** [comment]
@danielpataki there is a new version 1.1.0

**Pull request merged**